www/js/metrics.js

```diff
@@ -802,9 +802,9 @@ angular.module('emission.main.metrics',['nvd3', 'emission.services', 'ionic-date
                 // Here, we check if the string is all upper case by
                 // converting it to upper case and seeing if it is changed
                 if (field == field.toUpperCase()) {
-                    if (field === "WALKING" || field === "RUNNING") {
+                    // if (field === "WALKING" || field === "RUNNING") {
```

Comment:
this seems actively wrong when you remove the `if`. It looks like you always set the field to `ON_FOOT` no matter what the original field was (e.g. `CAR` will become `ON_FOOT`). what am I missing here?

Reply:
It **is** wrong 😮 I'm very sorry. My intention was to restore the original code... I'll correct that ASAP